src/components/Game.js

```diff
@@ -0,0 +1,78 @@
+import React, { Component } from 'react';
+
+import Button from './Button';
+import Input from './Input';
+
+class Game extends Component {
+    constructor(props) {
+        super(props);
+
+        this.onInputChange = this.onInputChange.bind(this);
+        this.onGo = this.onGo.bind(this);
+        this.onResetGame = this.onResetGame.bind(this);
+
+        this.state = {
+            roundName: 'round:',
+            round: 1,
+            numberToGuess: Math.floor((Math.random() * 100) + 1),
```

Comment:
`try to extract `Math.floor` to separate method of this class and invoke this method in both place you need.